### ISSUE TITLE
Passive Error with Google Chrome

### ISSUE THREAD

**Issue by BenDuh** [jquery/jquery-mousewheel#218]
Hi,

I'm using this function in my code to be able to switch vertical and horizontal scrolling :
```javascript
$("html, body, *")
    .off("mousewheel")
    .mousewheel(function (e, delta) {
         this.scrollTop -= delta / 2; //scrollLeft for horizontal
        //my code
   }
```
   
 I can't use {passive: true} with this function, can u explain what i did wrong ?

**Comment by mgol** [comment]
Duplicate of #202. Passive event handler support is blocked on jquery/jquery#2871.
